src/presentations/indexable-post-type-presentation.php

```diff
@@ -70,6 +86,43 @@ public function generate_og_type() {
 		return 'article';
 	}
 
+	/**
+	 * Generates the open graph article author.
+	 *
+	 * @return string The open graph article author.
+	 */
+	public function generate_og_article_author() {
+		$post = $this->post->get_post( $this->model->object_id );
+
+		if ( ! $post ) {
```

Comment:
I am not sure this is needed here. The check of the post existence should be done on a higher level.